indico/modules/events/papers/controllers/management.py

```diff
@@ -123,6 +125,23 @@ def _process(self):
                                               form=form)
 
 
+class RHManageContact(RHManagePapersBase):
+    """Allow to send emails to members of reviewing team"""
+    def _process(self):
+        cfp = self.event_new.cfp
+        paper_persons_dict = defaultdict(lambda: {'judge': False, 'content_reviewer': False, 'layout_reviewer': False})
+        for judge in cfp.judges:
```

Comment:
iterate over `self.event_new.acl_entries` and then check the roles like the methods in the CFP class do